eslint/eslint-common.json

```diff
@@ -0,0 +1,53 @@
+{
+    "extends": [
```

Comment:
I'm wondering if we can also have `airbnb-base` here for non-react projects? It would be useful for topology and ui-common. I don't know how using both: `airbnb-base` + `airbnb` behaves.

Reply:
After some hacking with extends order it turned out to be possible: `airbnb` extends `airbnb-base` so both can be used as long as order is maintained. Additionally, `prettier` needs to be placed after both as it disables some rules that `airbnb-base` enables.